reconcile/utils/git.py

```diff
@@ -6,9 +6,13 @@ class GitError(Exception):
     pass
 
 
-def clone(repo_url, wd):
+def clone(repo_url, wd, depth=None):
```

Comment:
```suggestion
def clone(repo_url, wd, depth=None):
    cmd = ["git", "clone"]
    if depth:
        cmd += ["--depth", str(depth)]
    cmd += [repo_url, wd]
```
something like this, would be easier to read